src/app/src/test/kotlin/org/imperial/mrc/hint/integration/ADRTests.kt

```diff
@@ -75,11 +75,8 @@ class ADRTests : SecureIntegrationTests()
         assertSuccess(result, null)
         var data = ObjectMapper().readTree(result.body!!)["data"]
         assertThat(data["pjnz"].textValue()).isEqualTo("inputs-unaids-spectrum-file")
-
-        result = testRestTemplate.getForEntity<String>("/adr/schemas/")
-        assertSuccess(result, null)
-        data = ObjectMapper().readTree(result.body!!)["data"]
-        assertThat(data["pjnz"].textValue()).isEqualTo("inputs-unaids-spectrum-file")
+        assertThat(data["outputZip"].textValue()).isEqualTo("inputs-unaids-naomi-output-zip")
```

Comment:
Not sure why this test was the same thing twice - fetching the result and checking the pjnz, but didn't seem necessary!